onnxruntime/core/mlas/lib/quantize.cpp

```diff
@@ -284,3 +284,139 @@ Return Value:
 {
     return MlasQuantizeLinearKernel<int8_t, -127, 127>(Input, Output, N, Scale, ZeroPoint);
 }
+
+#if defined(MLAS_SSE2_INTRINSICS)
+
+MLAS_FORCEINLINE
+MLAS_INT32X4
+MlasRequantizeOutputVector(
+    MLAS_INT32X4 IntegerVector,
+    MLAS_INT32X4 BiasVector,
+    MLAS_FLOAT32X4 ScaleVector,
+    MLAS_FLOAT32X4 MinimumValueVector,
+    MLAS_FLOAT32X4 MaximumValueVector,
+    MLAS_INT32X4 ZeroPointVector
+    )
+{
+    IntegerVector = _mm_add_epi32(IntegerVector, BiasVector);
+    MLAS_FLOAT32X4 FloatVector = _mm_cvtepi32_ps(IntegerVector);
+
+    //
+    // Scale the input vector and clamp the values to the minimum and maximum
+    // range (adjusted by the zero point value).
+    //
+
+    FloatVector = MlasMultiplyFloat32x4(FloatVector, ScaleVector);
+
+    // N.B. MINPS and MAXPS returns the value from the second vector if the
+    // value from the first vector is a NaN.
+    FloatVector = _mm_max_ps(FloatVector, MinimumValueVector);
+    FloatVector = _mm_min_ps(FloatVector, MaximumValueVector);
+
+    //
+    // Convert the float values to integer using "round to nearest even" and
+    // then shift the output range using the zero point value.
+    //
+
+    // N.B. Assumes MXCSR has been configured with the default rounding mode of
+    // "round to nearest even".
```

Comment:
I wonder if this belongs to the function doc header? In general, I think it would be better to put those in the headers. I wonder if it makes a difference for IDE when it displays the hoovering doc.

Reply:
The comment is just noting that the control register should be in the default ABI state used by all OS platforms. It's just there as a reminder that a mode sensitive instruction is okay to use here.